test/server/api/accounts.js

```diff
@@ -143,6 +143,27 @@ lab.experiment('Accounts Plugin Result List', () => {
             done();
         });
     });
+
```

Comment:
We can remove these tests. Since `pagedFind` is mocked (and we're not doing an integration test) we're not actually testing that special characters don't crash the app.